e2e/common.js

```diff
@@ -104,14 +106,12 @@ export const sendBitcoinTransaction = async(
 }
 
 export const returnBitcoinToDepositor = async (
-    web3,
```

Comment:
We are still passing this parameter in the client code.

Reply:
Yup, good catch. Removed